Source/WebKit/UIProcess/mac/WebViewImpl.mm

```diff
@@ -4671,6 +4671,11 @@ static NSPasteboardName pasteboardNameForAccessCategory(WebCore::DOMPasteAccessC
     [m_textAnimationTypeManager removeTextAnimationForAnimationID:uuid];
 }
 
+void WebViewImpl::hideTextAnimationView()
```

Comment:
should this maybe be `hideIntelligenceTextAnimationView`?

Reply:
It matches the other methods in this file. I could also imagine text animations / text effect views being used for non-intelligence purposes.